<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton_befor_KW">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../_Symbole/UsedSymbols/keyboard-left-arrow-button_icon-icons.com_72692.ico</normaloff>../../_Symbole/UsedSymbols/keyboard-left-arrow-button_icon-icons.com_72692.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="7">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget_Wochen_Plan">
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="autoScrollMargin">
        <number>16</number>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="rowCount">
        <number>12</number>
       </property>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>Montag</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Dienstag</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Mittwoch</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Donnerstag</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Freitag</string>
        </property>
       </column>
       <item row="7" column="3">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <pointsize>32</pointsize>
         </font>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_hauptmenue">
         <property name="text">
          <string>Hauptmen√º</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../_Symbole/Icons/computer/log_out.ico</normaloff>../../_Symbole/Icons/computer/log_out.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Kopieren</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../_Symbole/UsedSymbols/copyblackinterfacesymboloftwopapersheets_79854.ico</normaloff>../../_Symbole/UsedSymbols/copyblackinterfacesymboloftwopapersheets_79854.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_excel_ausgabe">
         <property name="text">
          <string>Excel ausgabe</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../_Symbole/UsedSymbols/microsoft_excel_icon_132212.ico</normaloff>../../_Symbole/UsedSymbols/microsoft_excel_icon_132212.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Hilfe">
         <property name="text">
          <string>Hilfe</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../_Symbole/UsedSymbols/info_13213.ico</normaloff>../../_Symbole/UsedSymbols/info_13213.ico</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="te_KW">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_jahr">
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Wochen Unterricht</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="pushButton_next_KW">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../_Symbole/UsedSymbols/keyboard-right-arrow-button-1_icon-icons.com_72690.ico</normaloff>../../_Symbole/UsedSymbols/keyboard-right-arrow-button-1_icon-icons.com_72690.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>KW</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Jahrl</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
